spec/models/account_spec.rb

```diff
@@ -1,34 +1,87 @@
 describe Account do
-  before(:each) do
-    @user = FactoryGirl.create(:account_user)
-    @group = FactoryGirl.create(:account_group)
+  let(:user) { FactoryGirl.create(:account_user) }
+  let(:group) { FactoryGirl.create(:account_group) }
```

Comment:
Let's use `user_account` and `group_account` respectively to refer to these accounts (you can't infer from the spec below that you aren't talking about literal users and groups). 

The fact that it's then `user_account` => `account_user` is simply because the factories these lines use are poorly constructed and should be rewritten with traits, but that's not necessary for this PR.